RELEASE_NOTES.md

```diff
@@ -4,6 +4,10 @@
 
 ...
 
+## 0.17.14 (2022-10-20)
+
+* Added support for Directory type outputs for WDL 2.0
```

Comment:
How about saying it's the `development` version (to match how we refer to this version in the Readme in dxCompiler as a version that we partially support)

Reply:
Here and also in `dxCompiler` release note please. Thanks!

Reply:
fixed in [72743b6](https://github.com/dnanexus/wdlTools/pull/236/commits/72743b623346b3b7534c9fbfe2455d23f7bf4325)